lib/memjs/utils.js

```diff
@@ -53,17 +55,19 @@ exports.parseMessage = function(dataBuf) {
   if (dataBuf.length < 24) {
     return false;
   }
-  var responseHeader = header.fromBuffer(dataBuf);
-  if (dataBuf.length < responseHeader.totalBodyLength + 24 || responseHeader.totalBodyLength < responseHeader.keyLength + responseHeader.extrasLength) {
+  var h = header.fromBuffer(dataBuf);
```

Comment:
Variables should be named meaningfully. If the issue is the if statement line length, break up the line instead